pandas/io/gbq.py

```diff
@@ -682,6 +692,15 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
         .. versionadded:: 0.19.0
 
+    **kwargs: Arbitrary keyword arguments
```

Comment:
Also, the ``**`` in the kwargs will give an error when building the docs (as this is used to make something bold). 

If you want to keep them, you need to make the docstring a raw string by adding a 'r' at the front (so ``r"""``)